UNIFY-328 Update all references to config file (#4093)
cypress-io/cypress-documentation: content/guides/core-concepts/writing-and-organizing-tests.md

the above structure.

You can modify the folder configuration in your configuration file. See

> [configuration file](/guides/references/configuration#Folders-Files) for more

Reviewer: ```suggestion
the [Cypress configuration](/guides/references/configuration#Folders-Files) for more
```

[`videosFolder`](/guides/references/configuration#Videos) to store the
screenshots and videos taken during the testing of your application. Many users
will opt to add these folders to their `.gitignore` file. Additionally, if you
are storing sensitive environment variables in your

> [configuration file](/guides/references/configuration) or

Reviewer: ```suggestion
[Cypress configuration](/guides/references/configuration) or
```

### Test Configuration

> To apply a specific Cypress
> [configuration file](/guides/references/configuration) value to a suite or test,

Reviewer: ```suggestion
To apply a specific
[Cypress configuration](/guides/references/configuration) value to a suite or test,
```

#### Files

> - [Configuration file](/guides/references/configuration)

Reviewer: ```suggestion
- [Cypress configuration](/guides/references/configuration)
```
